cmd/podman/common.go

```diff
@@ -197,6 +197,10 @@ var createFlags = []cli.Flag{
 		Name:  "hostname",
 		Usage: "Set container hostname",
 	},
+	cli.StringFlag{
+		Name:  "image-volumes",
+		Usage: "Image volume handling, either 'bind', 'tmpfs', or 'ignore' (default 'bind')",
```

Comment:
Should we call this builtin image volume?

Reply:
so the flag would be --builtin-image-volume? I feel like that is really long. If image-volume is not clear, we could call it --builtin-volume?

Reply:
I think having both `image-volume` and `builtin-volume` would be good.

Reply:
okay, it is now image-volume or builtin-volume.